src/app/components/random/random.component.html

```diff
@@ -0,0 +1,21 @@
+<button id="btnn" class="button" (click)="fetchRandomRecipe()">Obtener Receta Aleatoria</button>
+<div class="random-recipe">
+  <div *ngIf="randomRecipe.recipes && randomRecipe.recipes.length > 0" class="random-recipe__details">
+    <h2 class="random-recipe__title">{{ randomRecipe.recipes[0].title }}</h2>
+    <img [src]="randomRecipe.recipes[0].image" alt="Imagen de la receta" class="random-recipe__image">
+    <p [innerHTML]="randomRecipe.recipes[0].summary" class="random-recipe__summary"></p>
```

Comment:
```suggestion
    <p [innerText]="randomRecipe.recipes[0].summary" class="random-recipe__summary"></p>
```
No es mejor practica utilizar innerText?